http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -713,22 +717,21 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
     for {
       ctx <- ev.ask
 
-      // The "create app" flow does a number of things:
-      //  1. create, poll, and setup cluster if it doesn't exist
-      //  2. create and poll nodepool if it doesn't exist
-      //  3. create and poll disk if it doesn't exist
-      //  4. create and poll app
+      // The "create app" flow potentially does a number of things:
+      //  1. creates a Kubernetes cluster
```

Comment:
Doesn't the `if it doesn't exist` part still apply?

Reply:
Yeah, will add it back